### PR TITLE
EI-1051 Employer Withdrawal (Application Cancellation) email notification #347

### PR DESCRIPTION
**Pull request by ruslanss**
…quest

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Web/Controllers/CancelApprenticeshipsController.cs [hunk 1]
```diff
@@ -56,16 +57,21 @@ public async Task<IActionResult> Confirm(SelectApprenticeshipsRequest request)
...
 
-            var apprenticeshipIncentivesToWithdraw = (await GetSelectViewModel(request.AccountId, request.AccountLegalEntityId)).ApprenticeshipIncentives.Where(a => a.Selected);
+            var apprenticeshipIncentivesToWithdraw = 
+                (await GetSelectViewModel(request.AccountId, request.AccountLegalEntityId))
+                .ApprenticeshipIncentives.Where(a => a.Selected).ToList();
 
-            await _apprenticeshipIncentiveService.Cancel(request.AccountLegalEntityId, apprenticeshipIncentivesToWithdraw);
+            var emailAddress = ControllerContext.HttpContext.User.FindFirst(c => c.Type.Equals(EmployerClaimTypes.EmailAddress))?.Value;
+
+            await _apprenticeshipIncentiveService.Cancel(request.AccountLegalEntityId, apprenticeshipIncentivesToWithdraw,
```
Comment (David-B-Read) [inline src/SFA.DAS.EmployerIncentives.Web/Controllers/CancelApprenticeshipsController.cs:73]: The AC says that the email should be sent to the user that submitted the application for the cancelled apprentice - the logged in user might not be the person who originally submitted - worth clarifying

Comment (Stephek) [reply src/SFA.DAS.EmployerIncentives.Web/Controllers/CancelApprenticeshipsController.cs:73]: SubmittedByEmail is on the Application record.

Comment (Stephek) [reply src/SFA.DAS.EmployerIncentives.Web/Controllers/CancelApprenticeshipsController.cs:73]: What if they have left?

Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Web/Controllers/CancelApprenticeshipsController.cs:73]: It probably makes sense for the confirmation email to go to the person who actioned the cancellation, but the ticket doesn't state that at present.

Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Web/Controllers/CancelApprenticeshipsController.cs:73]: AC in tickets updated, approved

**Review by David-B-Read — APPROVED** [review]

**Pull request merged**